components/org.wso2.carbon.identity.application.authenticator.google/pom.xml

```diff
@@ -92,7 +96,12 @@
 
                             org.wso2.carbon.ui; version="${carbon.kernel.package.import.version.range}",
                             net.minidev.json; version="${net.minidev.json.imp.pkg.version.range}",
+                            org.wso2.carbon.idp.mgt; version="${carbon.identity.framework.package.import.version.range}",
```

Comment:
Don't we need `com.google.api.client.*` as an import package?

Reply:
Added as an Embed-Dependency